pages/tools/generator.js

```diff
@@ -19,12 +19,17 @@ export default function GeneratorPage() {
           icon={<IconRocket className="inline-block w-6 h-6 -mt-1" />}
           className="w-full mb-2 sm:w-auto sm:mb-0 sm:mr-2"
         /> */}
+        <div
+        className="flex"
+        >
+          
         <GithubButton
-          className="w-full sm:w-auto"
+          className="w-full sm:w-auto text-xs sm:text-lg flex justify-center items-center"
```

Comment:
Why different font size has been used for different viewpoints? It looks quite odd both in web and mobile view. Kindly don't change the internal properties of buttons.